mmcv/datasets/pipelines/transforms.py

```diff
@@ -0,0 +1,51 @@
+from collections.abc import Sequence
+
+import numpy as np
+
+import mmcv
+from ..builder import PIPELINES
+
+
+@PIPELINES.register_module()
+class Normalize:
+    """Normalize the image.
+
+    Added key is 'img_norm_cfg'.
```

Comment:
Should we add a docstring about "Required key" here?

Reply:
In fact, adding the docstring here will make the user confused. So I add it in __call__.

Reply:
Adding an example will be better